cbmc_viewer/optionst.py

```diff
@@ -315,6 +316,17 @@ def version(parser):
     )
     return parser
 
+def source_language(parser):
```

Comment:
What happens if we have mixed language code, e.g. Rust that calls C, which is then linked together.

Reply:
I'm not sure how to approach this. We could try to infer source language from the file extension, but if we want some way of explicitly passing it in, we'd need to pass in a mapping from file to language, which seems clunky.